packages/plugin-flex/src/commands/flex/plugins/upgrade-plugin.ts

```diff
@@ -101,6 +105,27 @@ export default class FlexPluginsUpgradePlugin extends FlexPlugin {
     flexPlugin,
   ];
 
+  private static packagesToRemoveFlexUI2 = {
+    'react-router': 'react-router-dom',
+    'react-router-redux': 'react-router-dom',
+    emotion: '@emotion/css',
+    'emotion-theming': '@emotion/react',
+    'react-emotion': '@emotion/react',
+    'create-emotion-styled': '@emotion/styled',
+  };
+
+  private static packageVersionsFlexUI2 = {
+    react: '^17.0.2',
+    'react-dom': '^17.0.2',
+    'react-redux': '^7.2.2',
+    redux: '^4.0.5',
+    'react-router-dom': '^5.2.0',
```

Comment:
None of these are in the original template, why are they here then?

Reply:
i was following the dependencies listed here https://docs.google.com/document/d/1f1NzVs3a_wANHrb2J1vswNXTZhc6uLkFnVCS9CEX8ow/edit#heading=h.bwfhq57luvu -- i interpreted these as if these dependencies exist in the users plugin (eg they installed the packages themselves) then they need to be upgraded otherwise flex ui 2.0 will break. is that not the case?

Reply:
ah good call.